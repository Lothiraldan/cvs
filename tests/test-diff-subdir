#!/bin/sh

hg init

mkdir alpha
touch alpha/one
mkdir beta
touch beta/two

hg add alpha/one beta/two
hg ci -m "start" -d "1000000 0"

echo 1 > alpha/one
echo 2 > beta/two

echo EVERYTHING
hg diff --nodates

echo BETA ONLY
hg diff --nodates beta

echo INSIDE BETA
cd beta
hg diff --nodates .
