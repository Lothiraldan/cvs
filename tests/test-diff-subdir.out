EVERYTHING
diff -r ec612a6291f1 alpha/one
--- a/alpha/one
+++ b/alpha/one
@@ -0,0 +1,1 @@
+1
diff -r ec612a6291f1 beta/two
--- a/beta/two
+++ b/beta/two
@@ -0,0 +1,1 @@
+2
BETA ONLY
diff -r ec612a6291f1 beta/two
--- a/beta/two
+++ b/beta/two
@@ -0,0 +1,1 @@
+2
INSIDE BETA
diff -r ec612a6291f1 beta/two
--- a/beta/two
+++ b/beta/two
@@ -0,0 +1,1 @@
+2
